irr/src/CGUIEditBox.cpp

```diff
@@ -1107,7 +1107,7 @@ void CGUIEditBox::breakText()
 				// TODO: I (Michael) think that we shouldn't change the text given by the user for whatever reason.
 				// Instead rework the cursor positioning to be able to handle this (but not in stable release
 				// branch as users might already expect this behavior).
-				Text.erase(i + 1);
+				Text.erase(i);
 				--size;
 				if (CursorPos > i)
```

Comment:
I think this has to be adjusted too, for correctness
```suggestion
				if (CursorPos >= i)
```

Reply:
You're right, done.